Foundation/FileHandle.swift

```diff
@@ -47,6 +60,79 @@ open class FileHandle : NSObject, NSSecureCoding {
     }
 
     internal func _readDataOfLength(_ length: Int, untilEOF: Bool, options: NSData.ReadingOptions = []) throws -> NSData.NSDataReadResult {
+#if os(Windows)
+        if _handle == INVALID_HANDLE_VALUE {
+          return NSData.NSDataReadResult(bytes: nil, length: 0, deallocator: nil)
+        }
```

Comment:
Should this not be `precondition(_handle != INVALID_HANDLE_VALUE, "Bad file handle")` to match the 
`precondition(_fd >= 0, "Bad file descriptor")` on the unix side?

Reply:
Huh?  I must've been asleep at the keyboard.